ci/build_ubuntu.sh

```
@@ -99,33 +105,48 @@ function prepare_env() {
 function save_integration_output() {
     echo Saving integration test output to ${INTEGRATION_OUTPUT}
-    local artifacts_dir=${ROOT_DIR}/.build-release${san_suffix}
+    local artifacts_dir=${ROOT_DIR}/.build-${BUILD_CONFIG}${san_suffix}
+    local third_party_dir=${ROOT_DIR}/.build-release${san_suffix}
     echo Saving build artifacts from ${artifacts_dir}
-    cp ${artifacts_dir}/valkey-json/build/src/libjson.so ${INTEGRATION_OUTPUT}
-    cp ${artifacts_dir}/valkey-server/.build-release/bin/valkey-server ${INTEGRATION_OUTPUT}
-    cp ${artifacts_dir}/libsearch.so ${INTEGRATION_OUTPUT}
-    local result_dir=${ROOT_DIR}/.build-release${san_suffix}/integration/.valkey-test-framework
+    if [ -f "${third_party_dir}/valkey-json/build/src/libjson.so" ]; then
+        cp ${third_party_dir}/valkey-json/build/src/libjson.so ${INTEGRATION_OUTPUT}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)